src/main/java/com/easypost/model/BaseCreditCard.java

```diff
@@ -1,125 +1,22 @@
 package com.easypost.model;
 
+import lombok.Getter;
+import lombok.Setter;
+
 /**
  * BaseCreditCard is a model class that represents the base of any credit card.
  *
  * @deprecated Use {@link com.easypost.model.PaymentMethodObject} instead.
  * Deprecated: v5.5.0 - v7.0.0
  */
 @Deprecated
+@Getter
 public class BaseCreditCard extends EasyPostResource {
-    private String object;
-    private String name;
+    private @Setter String brand;
+    private @Setter String expMonth;
+    private @Setter String expYear;
+    private @Setter String id;
+    private @Setter String object;
```

Comment:
Why do we have setters here?

I also thought we were going to remove the CreditCard class?

Reply:
`PrimaryPaymentMethod` and `SecondaryPaymentMethods` classes are inheriting the `BaseCreditCard` class, these two classes are deprecated but will not be removed until v7.0.0

Setters are needed for these variables because they are being used in here: https://github.com/EasyPost/easypost-java/blob/master/src/main/java/com/easypost/model/PaymentMethod.java#L24-L28

Reply:
Can we just start fresh with this rewrite and delete all deprecated functions?

Reply:
Hmmm, I can update the PR title